src/Boundary/IssueLocation/index.ts

```diff
@@ -0,0 +1,19 @@
+import { IIssueLocation } from "../../Domain/IssueLocation";
+
+export interface IIssueLocationGateway {
+    loadIssueLocations: () => Promise<ILoadIssuesOutputMeetingModel>;
+}
+
+export interface ILoadIssuesOutputMeetingModel{
+    successful: boolean,
+    issueLocations: IIssueLocation[]
+}
+
+export class LoadIssuesOutputMeetingModel implements ILoadIssuesOutputMeetingModel{
```

Comment:
Typically we don't put the actual implementation of the interfaces in the boundary, only the interfaces themselves. The class implementation should be alongside the use case (I don't think you have one) - the gateway shouldn't be returning this model, it should just be returning the data from the request.

Reply:
Ok, I'll move this into the UseCase folder.

Reply:
Resolved in https://github.com/LBHackney-IT/lbh-etra-front-end/pull/14/commits/263b6ebeef3305a9d94fb064a9a5564c6f7945a1